README.md

```diff
@@ -1,23 +1,32 @@
 # LorentzVectorHEP
 
 Provides two types (and the conversion between the two):
+
 - `LorentzVector` (energy, px, py, pz) (from [LorentzVectors.jl](https://github.com/JLTastet/LorentzVectors.jl))
 - `LorentzVectorCyl` (pt, eta, phi, mass)
 
 you can also use `fromPtEtaPhiE(pt, eta, phi, energy) --> LorentzVectorCyl`.
 
 and these functions for both of them:
+
 ```julia
-px, py, pz, energy, pt, eta, phi, mass
+px, py, pz, energy, pt, rap, eta, phi, mass
```

Comment:
I think we might just want to call this `rapidity`? that's what `ROOT` and scikit-hep `vector` calls it

Reply:
Good point - it's a better method name